test/01-constrain.test.ts

```diff
@@ -1,7 +1,10 @@
 import * as supertest from 'supertest';
+import axios from 'axios';
```

Comment:
Thanks, I removed it and am now using `superagent`. I can't use supertest because on this test step I'm validating that the file was actually stored on minio, so the test accesses the resource using the URL that points to minio, not an "app" URL as supertest uses

Reply:
@ramirogm I don't understand that you cannot use supertest. Please check me suggestion on the read file test case comparing the actual buffer.